### PR TITLE
{bio}[foss/2016b] HISAT2 v2.0.4

### PR DESCRIPTION
**Pull request by verdurin**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
@verdurin missing deps:

```
EasyBuildError: 'Irresolvable dependencies encountered: NGS/1.2.3-foss-2016b, ncbi-vdb/2.5.8-1-foss-2016b'
```

**Comment by boegel** [comment]
@verdurin the plot thickens:

```
ERROR: Irresolvable dependencies encountered: file/5.25-foss-2016b
```

BTW, did you check for more recent versions of these deps?

**Comment by verdurin** [comment]
As ever, your zeal for running the latest upstream holds us all to shame, @boegel. Just updated `file` and will look at the others.

File: easybuild/easyconfigs/n/ncbi-vdb/ncbi-vdb-2.5.8-1-foss-2016b.eb [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+ using data in the INSDC Sequence Read Archives."""
+
+toolchain = {'name': 'foss', 'version': '2016b'}
+
+source_urls = ['https://github.com/ncbi/ncbi-vdb/archive/']
+sources = ['%(version)s.tar.gz']
+
+dependencies = [
+    ('libxml2', '2.9.4'),
+    ('file', '5.25'),  # provides libmagic
```
Comment (boegel) [inline easybuild/easyconfigs/n/ncbi-vdb/ncbi-vdb-2.5.8-1-foss-2016b.eb]: @verdurin you'll need to bump this to `5.28` too

File: easybuild/easyconfigs/n/NGS/NGS-1.2.3-foss-2016b.eb [hunk 2]
```diff
@@ -0,0 +1,35 @@
+easyblock = 'ConfigureMake'
+
+name = 'NGS'
+version = '1.2.3'
```
Comment (boegel) [inline easybuild/easyconfigs/n/NGS/NGS-1.2.3-foss-2016b.eb]: @verdurin there's a 1.2.5 release of NGS

File: easybuild/easyconfigs/n/ncbi-vdb/ncbi-vdb-2.5.8-1-foss-2016b.eb [hunk 3]
```diff
@@ -0,0 +1,38 @@
+easyblock = 'ConfigureMake'
+
+name = 'ncbi-vdb'
+version = '2.5.8-1'
```
Comment (boegel) [inline easybuild/easyconfigs/n/ncbi-vdb/ncbi-vdb-2.5.8-1-foss-2016b.eb]: @verdurin this is at 2.7.0

File: easybuild/easyconfigs/n/ncbi-vdb/ncbi-vdb-2.5.8-1-foss-2016b.eb [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+
+toolchain = {'name': 'foss', 'version': '2016b'}
+
+source_urls = ['https://github.com/ncbi/ncbi-vdb/archive/']
+sources = ['%(version)s.tar.gz']
+
+dependencies = [
+    ('libxml2', '2.9.4'),
+    ('file', '5.28'),  # provides libmagic
+    ('HDF5', '1.8.17'),
```
Comment (boegel) [inline easybuild/easyconfigs/n/ncbi-vdb/ncbi-vdb-2.5.8-1-foss-2016b.eb]: there's an HDF5 `1.10.0-patch1` release, which is backwards compatible (I think), but not sure it's a good idea to pick that up as a dependency already...

**Comment by boegel** [comment]
lgtm

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 2 out of 6 (6 easyconfigs in this PR)
node2607.swalot.os - Linux centos linux 7.2.1511, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/038458cdf62c202ed3721a3056569b28 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 6 out of 6 (6 easyconfigs in this PR)
node2689.swalot.os - Linux centos linux 7.2.1511, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/b499b744713bc7ddc92b023f81913f53 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 6 out of 6 (6 easyconfigs in this PR)
node2081.delcatty.os - Linux SL 6.8, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/bbfbb922d193812426a07e19551a5825 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @verdurin!

**Pull request merged**